### PR TITLE
(PA-4191) Add Snyk monitoring

### PR DESCRIPTION
**Pull request by cthorn42**

### CODE REVIEW

File: .github/workflows/snyk_monitor.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+name: Snyk Monitor
+on:
+  schedule:
+    - cron: '0 5 * * 1-5'
```
Comment (joshcooper) [inline .github/workflows/snyk_monitor.yaml:5]: want to drop this part?

**Comment by joshcooper** [comment]
@cthorn42 one way to test the snyk monitor before merging is to push a feature branch to the puppetlabs/upstream repo instead of your fork. In doing so, your PR will have access to tokens (which still need to be added to this repo). That way we can validate that the snyk monitor check passes end-to-end before merging this PR.

**Comment by cthorn42** [comment]
Okay I need to do a squash, but I got this to work. Only trick was the name of the token/key was different. After updating that I was able to get the action to work, and on Snyk it shows that there is a report under puppet-foss there is a Hiera report.

**Comment by joshcooper** [comment]
Looks like the commit checker is failing. Maybe just prefix the commit summary with `(maint)`

**Review by joshcooper — APPROVED** [review]

**Pull request merged**